addon/components/utils/social-media-handle.ts

```diff
@@ -18,7 +18,7 @@ type SocialNetworkData = {
 
 export const SOCIAL_MEDIA_NETWORKS: SocialNetworkData[] = [
   { name: 'instagram', formattedUrl: 'https://www.instagram.com/{handle}' },
-  { name: 'twitter', formattedUrl: 'https://twitter.com/{handle}' },
+  { name: 'twitter', formattedUrl: 'https://x.com/{handle}' },
```

Comment:
should we add a row as a failsafe ?
```typescript
{ name: 'x', formattedUrl: 'https://x.com/{handle}' },
```

Reply:
We don't change the social media name in the backend. So, I don't think it is necessary